java/common/src/test/scala/org/typelevel/otel4s/java/InstancesSuite.scala

```diff
@@ -0,0 +1,15 @@
+package org.typelevel.otel4s.java
+
+import cats.effect.testkit.TestInstances
+import cats.effect.{IO, IOLocal}
+import cats.mtl.laws.discipline.LocalTests
+import munit.{CatsEffectSuite, DisciplineSuite}
+import org.scalacheck.Arbitrary.arbString
+import org.typelevel.otel4s.java.instances._
+
+class InstancesSuite extends CatsEffectSuite with DisciplineSuite with TestInstances {
+  implicit val ticker: Ticker = Ticker()
+  implicit val ioLocal: IOLocal[String] = IOLocal("").unsafeRunSync()(munitIORuntime)
```

Comment:
I'm not entirely sure if this is the best way to construct this, both `TestInstances` and `CatsEffectSuite` expose a `Runtime` so we have to pass it explicitly. Also, if I use the runtime from `TestInstances` the tests just hang.

Reply:
You can use `syncStep`.
https://github.com/typelevel/otel4s/blob/8c311369895f7e674444418b76ab23ada46d0cb0/java/common/src/main/scala/org/typelevel/otel4s/java/IOLocalContextStorageProvider.scala#L28-L38

Reply:
that does work too, but then i have to deal with an `Either`. I think I've found a nicer way where I don't even need `CatsEffectSuite`